### PR TITLE
Magazine cover - Luis Ruiz

### PR DESCRIPTION
**Pull request by luchodev**

### CODE REVIEW

File: bogota-abril-2019/css-magazine-cover/Luis-Ruiz/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            THE RISE & RISE OF
+          </h5>
+          <h2>
+            YCEE
+          </h2>
+        </div>
+      </header>
+      <section class="info">
+        <aside class="info-left">
+          <span>6</span> REASONS WHY GIRLS ARE CRAZY ABOUT RUNTOWN'S MAD OVER
```
Comment (gabrielmartinezglobant) [inline bogota-abril-2019/css-magazine-cover/Luis-Ruiz/index.html]: Unnecessary used of Capital letter.  Please use Text-transform CSS property.

https://css-tricks.com/almanac/properties/t/text-transform/

File: bogota-abril-2019/css-magazine-cover/Luis-Ruiz/index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            <h4 class="description">MUSIC VIDEOS OF 2016</h4>
+          </article>
+          <article class="info-right-anounce">
+            <h4 class="info-right-anounce-title">NEHRI OKWA</h4>
+            <h4 class="description">YOUNG GEO's MAKING IT</h4>
+          </article>
+        </aside>
+      </section>
+      <footer class="footer">
+        APPLE MUSIC LOCAL PRICING: A GOLDMINE FOR THE NEGERIAN MUSIC INDUSTRY
```
Comment (gabrielmartinezglobant) [inline bogota-abril-2019/css-magazine-cover/Luis-Ruiz/index.html]: Unnecessary used of Capital letter. Please use Text-transform CSS property.

https://css-tricks.com/almanac/properties/t/text-transform/

File: bogota-abril-2019/css-magazine-cover/Luis-Ruiz/index.html (hunk 1 above)
Comment (luchodev) [reply bogota-abril-2019/css-magazine-cover/Luis-Ruiz/index.html]: Ok, code updated

File: bogota-abril-2019/css-magazine-cover/Luis-Ruiz/index.html (hunk 2 above)
Comment (luchodev) [reply bogota-abril-2019/css-magazine-cover/Luis-Ruiz/index.html]: Code Updated

**Pull request merged**